upcloud/resource_upcloud_server.go

```diff
@@ -740,3 +759,11 @@ func resourceUpCloudServerDelete(ctx context.Context, d *schema.ResourceData, me
 
 	return diags
 }
+
+func cloudServerDefaultTitleFromHostname(hostname string) string {
+	suffix := " (managed by terraform)"
```

Comment:
Could this be a `const` above as well(?). Or is there a problem with placing the same const object in a bunch of strings 🤔